### PR TITLE
Adds support for Ctrl/Cmd-click on links to open in new tabs

### PR DESCRIPTION
**Pull request by raphaelhoffmann**

### CODE REVIEW

File: gui/frontend/src/search/search.coffee [hunk 1]
```diff
@@ -665,14 +666,35 @@ angular.module "mindbender.search", [
...
+                    newParams[k] = v
+                # update params 
+                newParams[qs] = params_qs
+                if @params.ro == 'true'
+                    newParams['ro'] = 'false'
+                arr = []
+                for k,v of newParams
+                    if v != null
+                        arr.push k + '=' + encodeURIComponent(v)
+                url = $location.protocol() + '://' + location.host + '/#' + $location.path() + '?' + arr.join('&')
```
Comment (alldefector) [inline gui/frontend/src/search/search.coffee]: Remove the protocol and host part. let the browser inherit it from current location.

Also, there should be some lib function to serialize the params, right? Say jQuery.param or something like that.

File: gui/frontend/src/search/search.coffee [hunk 2]
```diff
@@ -665,14 +666,35 @@ angular.module "mindbender.search", [
...
+                window.open(url, '_blank')
+            else
+                if @params.ro == 'true'
+                    @params.ro = 'false'
```
Comment (alldefector) [inline gui/frontend/src/search/search.coffee]: I'd leave the key `ro` out if value is false. It's cleaner.

File: gui/frontend/src/search/search.html [hunk 3]
```diff
@@ -238,7 +238,8 @@
                 </div>
                 <table ng-if="field !== 'images'" class="table table-hover table-condensed"
                     ng-init="fieldType = search.getFieldType(field)">
-                    <tr ng-repeat="bucket in agg.buckets track by $index" ng-click="search.doNavigate(field, bucket.key)">
+                    <!--<tr ng-repeat="bucket in agg.buckets track by $index" ng-click="search.doNavigate($event, field, bucket.key)">-->
+                    <tr ng-repeat="bucket in agg.buckets track by $index" ng-click="search.doNavigate($event, field, bucket.key)">
```
Comment (alldefector) [inline gui/frontend/src/search/search.html]: Is `$event` a builtin?

Comment (alldefector) [inline gui/frontend/src/search/search.html]: `doNavigate` is used in some app-specific templates. We need to update them as well.

Comment (raphaelhoffmann) [inline gui/frontend/src/search/search.html]: Yes, $event is an angular builtin. I updated calls to `doNavigate` in app-specific template as well.

**Pull request merged**